src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/DocumentController.kt

```diff
@@ -75,42 +75,37 @@ class DocumentController(
   fun getRecallDocument(
```

Comment:
Does it not need a `@ResponseStatus(HttpStatus.OK)` ?

Reply:
No because that's the default.  And the returned 200/OK for success is tested by the component test BUT it is hidden beyond the `authenticatedClient.getDocument` calls.